### DISCUSSION TITLE
OR I need to build the faust compiler with only the CPP backend

### DISCUSSION THREAD

**Discussion by bassmanitram** [grame-cncm/faust#1018] [Q&A] [answered]
...  but changing the backends CMAKE appears to building other embedded backends still. Again, any help would be appreciated

(Basically I want faust compiler to support ONLY CPP builds, but on a very old Linux system that doesn't support C++17 and/conversely doesn't have a GCC of the version when I build faust on a more modern Linux that does support C++17)

**Comment by sletz** [comment]
Read here: https://github.com/grame-cncm/faust/wiki/backends

**Reply by bassmanitram** [reply, marked as answer]
AH - need to do the make from the `build` dir - thx.
